sklearn/covariance/outlier_detection.py

```diff
@@ -62,9 +69,16 @@ def decision_function(self, X, raw_values=False):
             such as the One-Class SVM.
 
         """
+
+        if raw_values is not None:
+            warnings.warn(
+                'raw_values keyword has been moved to class initialization',
+                DeprecationWarning)
+            self.raw_values = raw_values
+
         check_is_fitted(self, 'threshold_')
         mahal_dist = self.mahalanobis(X)
-        if raw_values:
+        if self.raw_values:
             decision = mahal_dist
         else:
```

Comment:
I'm not a user of this class, but I find it a little obscure that the `raw_values=True` version is negatively correlated with the `raw_values=False` version. In a scoring context, it makes more sense to use negated distances. Should we change to that?